src/corva/handlers.py

```diff
@@ -55,11 +57,18 @@ def wrapper(aws_event: Any, aws_context: Any) -> List[Any]:
                     aws_event=aws_event, aws_context=aws_context
                 )
 
+                redis_client = redis.Redis.from_url(
+                    url=SETTINGS.CACHE_URL, decode_responses=True, max_connections=1
+                )
                 raw_events = raw_event_type.from_raw_event(event=aws_event)
 
                 results = [
                     func(
-                        raw_event, context.api_key, context.aws_request_id, logging_ctx
+                        raw_event,
+                        context.api_key,
+                        context.aws_request_id,
+                        logging_ctx,
+                        redis_client,
```

Comment:
We always pass redis_client to all functions, I guess its type hint should be `redis_client: redis.Redis` instead of `redis_client: Optional[redis.Redis]`